### ISSUE TITLE
Quick question about modifying the loss function

### ISSUE THREAD

**Issue by 7yzx** [isl-org/ZoeDepth#99]
Thank you for your great work !
However, as a beginner, I'm currently looking to obtain a depth map that emphasizes edge information more. I am considering modifying the loss function, such as adding additional gradient penalties and normal vector penalties. Unfortunately, my current hardware only allows for fine-tuning the metric head. In your opinion, is it feasible to achieve this by modifying the loss function within the constraints of fine-tuning the metric head? 
I greatly appreciate your help.
